java-checks/src/main/java/org/sonar/java/symexec/SymbolicEvaluator.java

```diff
@@ -370,177 +363,226 @@ public final void visitMemberSelectExpression(MemberSelectExpressionTree tree) {
       if (isSuperOrThisMemberSelect(tree)) {
         scan(tree.identifier());
       } else {
-        super.visitMemberSelectExpression(tree);
-        currentResult = SymbolicBooleanConstraint.UNKNOWN;
+        scan(tree.expression());
+        SymbolicValue value = new SymbolicValue();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this instantiation of "SymbolicValue". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2440)